crates/rolldown/tests/rolldown/topics/allow_different_variant_unit_test/_config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "config": {
+    "treeshake": false
+  },
+  "configVariants": [
+    {
+      "name": "enable-treeshake",
```

Comment:
Just a reminder, this option stands for 'OutputOptions.name'. 🫠

Reply:
I'm thinking about using $xx to represent non-bundle-options field but haven't thought it through yet.

Reply:
Then why did it passed into https://github.com/rolldown/rolldown/blob/6e9a9820fcad4cfcd543577373c7e680e8c1d659/crates/rolldown_testing/src/integration_test.rs?plain=1#L39

Reply:
I guess I add it for naming the test, but it now also got used as 'OutputOptions.name'.